_data/versions.yml

```diff
@@ -1,7 +1,10 @@
 v2.6:
-- title: v2.6.4
+- title: v2.6.5
   note: |
-    20 December 2017
+    22 December 2017
+
+    - Fixes rolling upgrades to v3.0 on clusters with etcd backend.
```

Comment:
Perhaps: Resolves issue which led to very brief losses of connectivity after upgrading to v3.0.0 from v2.6.5 when using an etcd datastore.